frontend/lib/src/components/shared/Radio/Radio.tsx

```diff
@@ -96,6 +121,8 @@ function Radio({
     return spacer ? "&nbsp;" : caption
   }
 
+  const [checkedRadioInnerSize, radioInnerSize] = getRadioInnerSizes(theme)
```

Comment:
Maybe we can put this into a simple memo with `theme` as a dependency (ideally, the actual theme values, but I think just doing a reference dependency check might be fine here).